arrow-libs/core/arrow-annotations/src/main/java/arrow/extension.kt

```diff
@@ -7,7 +7,6 @@ package arrow
   AnnotationTarget.FUNCTION
 )
 @MustBeDocumented
-@Deprecated(KindDeprecation)
 annotation class extension
```

Comment:
This and `higherkind` got undeprecated?

Reply:
Twice in a row 🤦 3 times and I owe you a beer!

Reply:
😘